provider/devices/common.go

```diff
@@ -995,3 +1006,25 @@ func applyDeviceStreamSettings(device *models.Device) error {
 
 	return nil
 }
+
+func updateGadsStreamSettings(device *models.Device) error {
```

Comment:
Can you move this function to `android.go` and export it so it can be reused by the [UpdateDeviceStreamSettings](https://github.com/shamanec/GADS/blob/99315b781c5ed44bfae90ca7f3f8cb790dfc3a24/provider/router/routes.go#L349C6-L349C32) endpoint?

Reply:
In general this is stupid but it was the fastest way to implement stream quality updates along with WebDriverAgent. Probably a normal http server should be added to GADS-Stream so we can just fire a request without multiple hoops for the stupid websocket 😅

Reply:
Done